thirdeye/thirdeye-frontend/app/pods/manage/alert/edit/controller.js

```diff
@@ -542,11 +506,11 @@ export default Controller.extend({
       });
 
       // Exit quietly (showing warning) in the event of error
-      if (emailError || this.get('isDuplicateEmail')) { return; }
+      if (emailError || isDuplicateEmail) { return; }
 
       // Assign these fresh editable values to the Alert object currently being edited
-      Ember.set(postFunctionBody, 'functionName', this.get('alertFunctionName'));
-      Ember.set(postFunctionBody, 'isActive', this.get('isActive'));
+      Ember.set(postFunctionBody, 'functionName', alertFunctionName);
```

Comment:
Perhaps we should use `Ember.run` here to make sure that the template only rerenders once for both `Ember.set`
https://guides.emberjs.com/v2.11.0/applications/run-loop/#toc_how-do-i-tell-ember-to-start-a-run-loop

Reply:
Actually, `postFunctionBody` is not being used by the template anywhere, nor is `alertData`, its source object.